File: Array/sorting/BubbleSort.py
def bubble_sort(arr):
    for i in range(len(arr)):
        for j in range(0, len(arr) - 1):
            if(arr[j] > arr[j + 1]):
                arr[j], arr[j + 1] = arr[j + 1], arr[j]

    for values in arr:
        print(values, end=" ")

# Example usage
if __name__ == "__main__":
    arr = [64, 34, 25, 12, 22, 11, 90]
    bubble_sort(arr)
    print("Sorted array is:", arr)

File: Array/sorting/BubbleSort.go
package main

import "fmt"

func bubbleSort(arr []int) {
	for i := 0; i < len(arr); i++ {
		for j := 0; j < len(arr) - 1; j++ {
			if arr[j] > arr[j+1] {
				arr[j], arr[j+1] = arr[j+1], arr[j]
			}
		}
	}

	for _, value := range arr {
		fmt.Print(value, " ")
	}
}

func bubbleSortMain() {
	arr := []int{2, 1, 3, 5, 4}
	bubbleSort(arr)
}